client/src/actions/images.js

```diff
@@ -0,0 +1,26 @@
+const ENDPOINT = '/api'
+
+const IMAGE_ENDPOINT = ENDPOINT + '/images'
+
+export const getImages = async (selected) => {
+    const {sharks, cats} = selected
+
+    let url = IMAGE_ENDPOINT
+
+    if (sharks && !cats) {
+        url += '?content=sharks'
+    } else if (!sharks && cats) {
+        url += '?content=cats'
+    }
```

Comment:
What if you had more than two categories? How would you handle this?

Reply:
In that case, I would create a helper function to turn the selected categories into a URL. This could be accomplished by mapping across the entries of the `selected` object and adding a `&content=${category}` for each selected category after the first. 

Note that this change would also require an update to the API in order to convert multiple parameter keys into a list of categories requested.